eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/steps/processors/VariantToVariantContextProcessor.java

```diff
@@ -58,27 +67,61 @@ public VariantContext process(IVariant variant) {
                 .id(variant.getMainId())
                 .source(variant.getMainId())
                 .alleles(allelesArray)
-                .attributes(getUniqueAttributes(variant))
+                .attributes(getAttributes(variant))
                 .unfiltered()
                 .make();
 
         return variantContext;
     }
 
-    private Map<String, String> getUniqueAttributes(IVariant variant) {
-        Map<String, Set<String>> attributes = new HashMap<>();
+    private Map<String, String> getAttributes(IVariant variant) {
+        Map<String, List<String>> attributesList = getAttributesList(variant);
+
+        Map<String, String> attributes = new HashMap<>();
+        for (Map.Entry<String, List<String>> attribute : attributesList.entrySet()) {
+            if (attribute.getKey().equals(VARIANT_CLASS_KEY)) {
+                attributes.put(VARIANT_CLASS_KEY, toUniqueConcatenation(attribute.getValue()));
+            } else if (attribute.getKey().equals(STUDY_ID_KEY)) {
+                attributes.put(STUDY_ID_KEY, toUniqueConcatenation(attribute.getValue()));
+            } else if (attribute.getKey().equals(ALLELES_MATCH_KEY)) {
+                if (attribute.getValue().stream().anyMatch(Boolean.toString(false)::equals)) {
+                    attributes.put(ALLELES_MATCH_KEY, "");
+                }
+            } else if (attribute.getKey().equals(ASSEMBLY_MATCH_KEY)) {
+                if (attribute.getValue().stream().anyMatch(Boolean.toString(false)::equals)) {
+                    attributes.put(ASSEMBLY_MATCH_KEY, "");
+                }
+            } else if (attribute.getKey().equals(SUPPORTED_BY_EVIDENCE_KEY)) {
+                if (attribute.getValue().stream().anyMatch(Boolean.toString(false)::equals)) {
+                    attributes.put(SUPPORTED_BY_EVIDENCE_KEY, "");
+                }
+            } else if (attribute.getKey().equals(VALIDATED_KEY)) {
+                if (attribute.getValue().stream().anyMatch(Boolean.toString(true)::equals)) {
+                    attributes.put(VALIDATED_KEY, "");
+                }
+            } else if (attribute.getKey().equals(SUBMITTED_VARIANT_VALIDATED_KEY)) {
+                long count = attribute.getValue().stream().filter(Boolean.toString(true)::equals).count();
+                attributes.put(SUBMITTED_VARIANT_VALIDATED_KEY, Long.toString(count));
+            }
```

Comment:
should we put a default case for other keys? putting all the unique values

Reply:
given that we want to have full control over the written info, I'm going to write only those fields.
Also, given that some of the fields might not be present, it might be a bit more verbose not to use the loop, so I'm keeping the switch